src/composables/ViewRoutingSlip/useTransactionDataTable.ts

```diff
@@ -0,0 +1,79 @@
+import { Invoice, InvoiceDto, LineItem, Reference } from '@/models/Invoice'
+import { ref, toRefs, watch } from '@vue/composition-api'
+
+import { createNamespacedHelpers } from 'vuex-composition-helpers'
+
+const routingSlipModule = createNamespacedHelpers('routingSlip') // specific module name
+const { useGetters } = routingSlipModule
+
+// Composable function to inject Props, options and values to TransactionDataTable component
+export default function useTransactionDataTable (props, _) {
+  // ref and i/p variables
+  const { invoices } = toRefs(props)
+  const invoiceDto = ref<InvoiceDto[]>([])
+  const headerTranscations = [
+    {
+      text: 'Date',
+      align: 'start',
+      sortable: false,
+      value: 'createdOn'
+    },
+    {
+      text: 'Invoice #',
+      align: 'start',
+      sortable: false,
+      value: 'invoiceNumber'
+    },
+    {
+      text: 'Transaction Amount',
+      align: 'end',
+      sortable: false,
+      value: 'total'
+    },
+    {
+      text: 'Description',
+      align: 'start',
+      value: 'description',
+      sortable: false
+    },
+    {
+      text: 'Initiator',
+      align: 'start',
+      value: 'createdName',
+      sortable: false
+    }
+  ]
+
+  // vuex getter
+  const { invoiceCount } = useGetters(['invoiceCount'])
+
+  watch(invoices, () => {
+    transformInvoices()
+  }, { deep: true, immediate: true })
+
+  function transformInvoices (): void {
+    for (let i = 0; i < invoices.value?.length; i++) {
+      const invoice: Invoice = invoices.value[i]
+      const invoiceDtoObject: InvoiceDto = {}
+      // assign created on date
+      invoiceDtoObject.createdOn = invoice.createdOn
+      // concatenate all description from line items
+      const description = invoice?.lineItems.map((lineItem: LineItem) => {
+        return lineItem?.description
```

Comment:
description need to show in one line?

Reply:
![image](https://user-images.githubusercontent.com/78374865/126679383-ea9be622-3d25-4eb5-9585-7842054039aa.png)
We are concatenating the description and displaying in new lines as per the picture